pkg/kubeapiserver/authenticator/config.go

```diff
@@ -181,7 +181,7 @@ func (config AuthenticatorConfig) New() (authenticator.Request, *spec.SecurityDe
 		if config.TokenSuccessCacheTTL > 0 || config.TokenFailureCacheTTL > 0 {
 			tokenAuth = tokencache.New(tokenAuth, config.TokenSuccessCacheTTL, config.TokenFailureCacheTTL)
 		}
-		authenticators = append(authenticators, bearertoken.New(tokenAuth), websocket.NewProtocolAuthenticator(tokenAuth))
+		authenticators = append(authenticators, bearertoken.New(tokenAuth), audunaware.WrapRequest(config.ServiceAccountAPIAudiences, websocket.NewProtocolAuthenticator(tokenAuth)))
```

Comment:
drop this change. this is wrapping the union of all token authenticators, which isn't right. we already wrapped the individual ones that required it earlier.